src/steps/devices/index.ts

```diff
@@ -511,6 +514,30 @@ export async function fetchComputers({
 
       computerDeviceIdToGraphObjectKeyMap.set(computer.id, computerEntity._key);
 
+      // build computer group and group relationship
+      for (const groupName of computerDetail.groups_accounts
+        .computer_group_memberships) {
```

Comment:
```suggestion
        ?.computer_group_memberships) {
```
Are groups_accounts always not undefined?